### ISSUE TITLE
Insufficient readme, what "application type" to choose in google cloud ?

### ISSUE THREAD

**Issue by reinardhz** [jay0lee/got-your-back#276]
Full steps to reproduce the issue:
1. Using gyb unix executable on macbook, download from `gyb-1.37-macos-x86_64-MacOS10.13.6.tar.xz`
2. Execute command: 
```
Administrators-MacBook-Pro-6:gyb user$ ./gyb --email somebody@gmail.com --action estimate
ERROR: Please configure a project

To make GYB run you will need to populate the client_secrets.json file. Try
running:

./gyb --action create-project --email somebody@gmail.com


[Errno 2] No such file or directory: '/Users/user/Downloads/gyb/client_secrets.json'Administrators-MacBook-Pro-6:gyb user$
```
3. Execute command:

```
Administrators-MacBook-Pro-6:gyb user$ ./gyb --action create-project --email somebody@gmail.com

Go to the following link in your browser:

	https://gyb-shortn.jaylee.us/hv2xxx

Enter verification code: 
```
4. Go to the link provided, click to allow "GAM Project Creation" make changes in my gmail account.
5. Verfication code show up, i copy that
6. Back to my terminal, i put the verification code, then press enter:
```
Enter verification code: 4/0gEiu-XzqeO1wAF5qxxxxxxxxxCPIPcxKk9YYa2SPuclPQVEQRvoq8
Creating project "Got Your Back Project"...
Checking project status...
 enabling API drive.googleapis.com...
 enabling API gmail.googleapis.com...
 enabling API groupsmigration.googleapis.com...
 enabling API vault.googleapis.com...
Creating Service Account
Please go to:

https://console.developers.google.com/apis/credentials/consent/edit?createClient&newAppInternalUser=true&project=gyb-project-xxx-xxx-xxx

1. Enter "GYB" for "Application name".
2. Leave other fields blank. Click "Save" button.
3. Choose "Other". Enter a desired value for "Name". Click the blue "Create" button.
4. Copy your "client ID" value.

Enter your Client ID:
```
7. Open the link in browser, Enter "GYB" for "Application name". Leave other fields blank. Click "Save" button.
8. Then this page show up:

<a href="https://drive.google.com/file/d/1XtOmLb6lf9Bhr80ZT-HHgdRKXcgEsdJo/view?usp=sharing">https://drive.google.com/file/d/1XtOmLb6lf9Bhr80ZT-HHgdRKXcgEsdJo/view?usp=sharing</a>

9. What to choose in the "Application Type" ? Please put more details in the wiki. Thanks.

**Comment by musq** [comment]
We need to choose `Desktop app` as the Application Type during creating an OAuth Client ID.
https://github.com/jay0lee/got-your-back/issues/277#issuecomment-645040380

**Comment by victorhooi** [comment]
We hit this issue yesterday as well.

GYB asks you to choose "Other" as the OAuth Client Type - but there is no such option:

<img width="602" alt="Screen Shot 2020-06-24 at 2 26 13 pm" src="https://user-images.githubusercontent.com/243499/85616018-82944880-b6a0-11ea-96dc-556de971c606.png">

If you pick "Web application", which seems a reasonable guess, it will fail - later on it will say "Error 400: redirect_uri_mismatch":

<img width="548" alt="Screen Shot 2020-06-24 at 2 28 10 pm" src="https://user-images.githubusercontent.com/243499/85616118-9d66bd00-b6a0-11ea-8eb0-16191611dc30.png">

Hence, I think the code should be updated to change from "Other" to "Desktop app".

**Comment by gene1wood** [comment]
This was also reported in #272 and fixed in #273
